lib/grinder_tools.dart

```diff
@@ -139,10 +139,10 @@ Future runProcessAsync(GrinderContext context, String executable,
  * A default implementation of an `init` task. This task verifies that the grind
  * script is executed from the project root.
  */
-void defaultInit(GrinderContext context) {
+void defaultInit([GrinderContext context]) {
```

Comment:
We can doc the param as deprecated. There are unfortunately many API methods which take `GrinderContext` as their first parameter. This means we can't change it to an optional param. We can either:

- keep the param, but no longer refer to it in the method, or
- remove the param completely. It would be a breaking change from the `0.6.0` version -